pkg/apis/serving/v1/configuration_validation.go

```diff
@@ -76,7 +77,7 @@ func (c *Configuration) validateLabels() (errs *apis.FieldError) {
 		switch key {
 		case serving.RouteLabelKey:
 			// Known valid labels.
-		case serving.VisibilityLabelKey:
+		case network.VisibilityLabelKey, serving.VisibilityLabelKeyObsolete:
```

Comment:
not specific to this PR, but I am not sure why we want to validate this label for configuration. I thought this label is only for Route.

Reply:
Yes, this validation check is not necessary in `configuration`. I removed them.